plugins/src/webrtcsink/imp.rs

```diff
@@ -1729,10 +1751,77 @@ impl WebRTCSink {
             webrtcbin.set_property("turn-server", turn_server);
         }
 
+        match settings.cc_info.heuristic {
+            WebRTCSinkCongestionControl::GoogleCongestionControl => {
+                let cc_info = settings.cc_info;
+                webrtcbin.connect_closure(
+                    "request-aux-sender",
+                    false,
+                    glib::closure!(@watch element, @strong peer_id
+                            => move |_webrtcbin: gst::Element, _transport: gst::Object| {
+
+                        let cc = match cc_info.heuristic {
+                            WebRTCSinkCongestionControl::GoogleCongestionControl => {
```

Comment:
Why do you need to match here? The whole closure is only connected if `cc_info` is gcc isn't it?

Reply:
For futureproofness I guess, I could remove it.

Reply:
please do yes :)